### ISSUE TITLE
Update Fehler

### ISSUE THREAD

**Issue by joogi26** [jgehrcke/covid-19-germany-gae#2413]
Hallo,
es gibt keine Updates mehr ?
Unter Actions sind nur Fehler zu sehen.
Letztes RKI Update 17.02.2022
Letztes Crowdsource Update 18.02.2022

**Comment by EmTee70** [comment]
Ja, es wäre großartig, wenn es bald wieder neue Daten gäbe! Vielen Dank jedenfalls für die bisher geleistete Arbeit!

**Comment by jgehrcke** [comment]
Thank you for the feedback! I've been offline/vacationing -- so I did not
notice. Will look into debugging soon, hopefully tomorrow. Sorry for the
delay!

**Comment by jgehrcke** [comment]
quick update: we're affected by this issue: https://github.com/pallets/markupsafe/issues/284

**Comment by EmTee70** [comment]
Blöder Mist, wenn man unter dem Hintern das funktionierende Release weggezogen bekommt... kenne mich mit Python leider gar nicht aus. Immerhin gibt es Hinweise, wie man die Version "künstlich" festsetzt. Wünsche (uns) viel Erfolg!

**Comment by jgehrcke** [comment]
fixed with https://github.com/jgehrcke/covid-19-germany-gae/pull/2414 :) -- manually triggered update was successful!

**Comment by joogi26** [comment]
Super dass es wieder läuft
Danke !
